claude/skills/jira/scripts/fetch-ticket.py

```
@@ -0,0 +1,412 @@
+        return None
+    except netrc.NetrcParseError:
+        return None
+    return None
+
+
+def diagnose_auth_error(config, error_code):
+    """Diagnose authentication failure and print guidance."""
+    jira_host = get_jira_host(config)
+
+    if error_code == 403:
+        print(f"ERROR 403: Access denied. CAPTCHA may be triggered.")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)